server/game/CardSelectors/BaseCardSelector.js

```diff
@@ -32,7 +32,7 @@ class BaseCardSelector {
         return context.game.allCards.filter(card => this.canTarget(card, context));
     }
 
-    hasEnoughSelected(selectedCards) {
+    hasEnoughSelected(selectedCards, numPlayers) { // eslint-disable-line no-unused-vars
```

Comment:
Not super keen on these placeholder variables that do nothing

Reply:
I think it's appropriate for base classes (which this is) to better document parameters that while not used here can be used in inheriting classes. Could be removed, but then should be documented in some other way.

Reply:
Probably makes more sense to use comments instead.  What we really need is abstract methods, but those don't exist.  Ideally we should also be "throw"'ing in base methods to make it more obvious that that's what this is